examples/references.bib

```diff
@@ -366,3 +366,10 @@ @book{wilkinson2005grammar
   issn={1431-8784},
   isbn={978-0-387-24544-7}
 }
+
+@book{roback2021beyond,
+  title={Beyond multiple linear regression: Applied generalized linear models and multilevel models in R},
+  author={Roback, P., and Legler, J.},
+  year={2021},
+  publisher={CRC Press}
```

Comment:
you can also add a `url` field to include https://bookdown.org/roback/bookdown-BeyondMLR/ in the bibliography citation too.

Reply:
Done :)